src/app/modules/group/pages/group-details/group-details.component.html

```diff
@@ -1,4 +1,4 @@
-<div *ngIf="state === 'loaded' && group"><!-- && group is used for asserting that the variable is set -->
+<div *ngIf="state === 'loaded' && (group$ | async) as group">
```

Comment:
Maybe remove parentheses around `(group$ | async)` to be consistent with the other `group$ | async as group`

Reply:
As there is this &&, it is mandatory, otherwise it applies "async" on `loaded' && group`